docker-compose.prod.yml

```diff
@@ -25,22 +25,22 @@ services:
       - geodjango_project_network_prod
     volumes:
       - static_volume:/home/app/web/staticfiles
-    expose:
-      - 8000
+    ports:
+      - 8000:8000
     depends_on:
       - postgis
 
   postgis:
-    image: postgis/postgis:latest
+    image: tobi312/rpi-postgresql-postgis:15-3.4-alpine-arm
     container_name: geodjango_postgis_prod
     restart: always
     healthcheck:
       test: ["CMD-SHELL", "pg_isready -U postgres"]
       interval: 10s
       retries: 5
     environment:
-      - POSTGRES_DB
-      - POSTGRES_PASSWORD
+      - POSTGRES_DBNAME
```

Comment:
kartoza remnant?

Reply:
@dheerajchand I think this is working because POSTGRES_DB is optional.
This image is a port of the official docker PostGIS image, and their docs say to use `POSTGRES_DB`.

https://hub.docker.com/r/postgis/postgis

I suggest just deleting it so someone doesn't try to use it and wonder why it has no effect.